CefSharp.Wpf/Experimental/WpfIMEKeyboardHandler.cs

```diff
@@ -198,7 +222,8 @@ private void OnImeComposition(IntPtr hwnd, int lParam)
 
                 if (ImeHandler.GetComposition(hwnd, (uint)lParam, underlines, ref compositionStart, out text))
                 {
-                    owner.GetBrowserHost().ImeSetComposition(text, underlines.ToArray(), new Range(int.MaxValue, int.MaxValue), new Range(compositionStart, compositionStart));
```

Comment:
I made that formatting change because the width was over the 132 column limit as mentioned in CONTRIBUTING.md. Let me know if you'd still like me to revert.

Could the .editorconfig also have the column width set in there? I imagine that's hard to enforce.

Reply:
Understood,  thanks.  Just leave it for now. 

Bundling formatting fixes and bug fixes in the same commit just makes it harder to review,  specifically ones like this.

I'm not sure how editorconfig handles wrapping,  probably worth investigating.